tools/proofers/proof_frame_enh.inc

```diff
@@ -13,6 +13,8 @@ function echo_proof_frame_enh( $ppage )
     global $code_url, $userP;
 
     $json_code_points = get_json_codepoints($ppage->projectid());
+    $switch_confirm = javascript_safe(_('Are you sure you want to change layout & save the current document?'));
```

Comment:
I hate that this confirmation message exists. But it does, so that's fine. Can we reorder the wording to put the important part the user might care about confirming first?

`Are you sure you want to save the current page and change layout?`

Reply:
Done this.